esp-hal/src/lib.rs

```diff
@@ -32,7 +32,7 @@
 //! Invoke the following command in the root of the esp-hal repository to get
 //! started.
 //!
-//! ```no_run
+//! ```ignore
```

Comment:
we probably should use `bash` as the language - rendering this as Rust might not look good?